proxy/src/auth/credentials.rs

```diff
@@ -68,11 +79,17 @@ impl TryFrom<HashMap<String, String>> for ClientCredentials {
 
         let user = get_param("user")?;
         let dbname = get_param("database")?;
+        let project_name = get_param("project");
+        let project_name = match project_name {
+            Ok(project_name) => Some(project_name),
+            Err(_) => None,
+        };
```

Comment:
See https://doc.rust-lang.org/std/result/enum.Result.html#method.ok

Reply:
thanks, just learning this.